### ISSUE TITLE
how to use avoid downloading unzip?

### ISSUE THREAD

**Issue by rapenne-s** [lutris/lutris#4190]
### Bug description

I'm using lutris on NixOS, that linux distribution is quite special with regards to library path and when I install some Linux games from GOG (like Tangledeep), lutris will download a file unzip.tar.gz along with the game installer, this will result in an installation failure because the downloaded `unzip` binary can't find libbz2.so.1.0 in the system.

Is it possible to make Lutris use `unzip` if found in $PATH ?

### How to Reproduce

1. on NixOS with lutris 0.5.10 , install a Linux game from GOG that requires unzip.tar.gz file
2. See the install fail because of unzip missing shared libraries

### Expected behavior

I see 3 options:

- downloaded `unzip` could be statically compiled to fix that issue and be sure it will work everywhere
- avoid downloading an `unzip` binary if lutris can find one in $PATH
- add a parameter to hardcode `unzip` path (I think this is not a really good idea)

### Log output

```shell
$ /nix/store/89a3m2zlbxhqqx22zpdpa6vfc4alp3pi-lutris/bin/lutris -d

INFO     2022-04-03 23:52:20,129 [startup.init_lutris:173]:Starting Lutris 0.5.10
WARNING  2022-04-03 23:52:20,160 [libretro.get_libretro_cores:24]:No folder at /home/jeux/.local/share/lutris/runners/retroarch/
DEBUG    2022-04-03 23:52:20,167 [xrandr._get_vidmodes:15]:Retrieving video modes from XrandR
INFO     2022-04-03 23:52:20,359 [startup.check_driver:65]:Running Intel Mesa driver 21.3.7 on Mesa Intel(R) HD Graphics 620 (KBL GT2) (0x5916)
INFO     2022-04-03 23:52:20,359 [startup.check_driver:77]:GPU: 8086:5916 17AA:2245 (i915 drivers)
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

INFO     2022-04-03 23:52:20,415 [startup.update_runtime:213]:Startup complete
DEBUG    2022-04-03 23:52:20,618 [lutriswindow.update_store:437]:Showing XX games
DEBUG    2022-04-03 23:52:23,650 [lutriswindow.update_store:437]:Showing XX games
DEBUG    2022-04-03 23:52:23,934 [lutriswindow.update_store:437]:Showing X games
DEBUG    2022-04-03 23:52:24,244 [lutriswindow.update_store:437]:Showing X games
DEBUG    2022-04-03 23:52:25,912 [base.install:210]:Installing 1703478259 from service gog
DEBUG    2022-04-03 23:52:25,913 [http.get:83]:GET https://lutris.net/api/games/service/gog
DEBUG    2022-04-03 23:52:26,124 [api.get_game_installers:160]:Fetching installer https://lutris.net/api/installers/tangledeep
DEBUG    2022-04-03 23:52:26,124 [http.get:83]:GET https://lutris.net/api/installers/tangledeep
DEBUG    2022-04-03 23:52:26,426 [application.show_window:332]:Showing window InstallerWindow1703478259
DEBUG    2022-04-03 23:52:27,394 [xrandr._get_vidmodes:15]:Retrieving video modes from XrandR
mount: /nix/store/yx5zifiwsc7zbd54zs7l8i4a6nrrg6wf-wrapper.c:195: main: Assertion `!(st.st_mode & S_ISUID) || (st.st_uid == geteuid())' failed.
ERROR    2022-04-03 23:52:27,661 [system.read_process_output:100]:['mount', '-v'] command failed: Command '['mount', '-v']' died with <Signals.SIGABRT: 6>.
DEBUG    2022-04-03 23:52:28,579 [interpreter.get_extras:159]:Checking if service provide extra files
DEBUG    2022-04-03 23:52:28,580 [gog.get_extras:326]:Download extras for GOG ID 1703478259 and its DLCs
INFO     2022-04-03 23:52:28,580 [gog.get_game_details:292]:Getting game details for 1703478259
DEBUG    2022-04-03 23:52:28,586 [http.get:83]:GET https://api.gog.com/products/1703478259?expand=downloads&locale=fr-FR
INFO     2022-04-03 23:52:29,049 [gog.get_game_details:292]:Getting game details for 1703478259
DEBUG    2022-04-03 23:52:29,055 [http.get:83]:GET https://api.gog.com/products/1703478259?expand=downloads&locale=fr-FR
DEBUG    2022-04-03 23:52:29,232 [http.get:83]:GET https://api.gog.com/products?HIDDENHERE
DEBUG    2022-04-03 23:52:29,545 [interpreter.create_game_folder:181]:Creating destination path /home/jeux/GOG Games/gog/tangledeep
DEBUG    2022-04-03 23:52:30,420 [http.get:83]:GET https://embed.gog.com/userData.json
INFO     2022-04-03 23:52:30,891 [installer.prepare_game_files:155]:Getting files for game
INFO     2022-04-03 23:52:30,892 [installer.prepare_game_files:157]:Adding selected extras to downloads
INFO     2022-04-03 23:52:30,892 [gog.get_game_details:292]:Getting game details for 1703478259
DEBUG    2022-04-03 23:52:30,897 [http.get:83]:GET https://api.gog.com/products/1703478259?expand=downloads&locale=fr-FR
INFO     2022-04-03 23:52:31,076 [gog.get_download_info:298]:Getting download info for https://api.gog.com/products/1703478259/downlink/installer/en3installer0
DEBUG    2022-04-03 23:52:31,078 [http.get:83]:GET https://api.gog.com/products/1703478259/downlink/installer/en3installer0
DEBUG    2022-04-03 23:52:32,069 [files_box.on_file_available:93]:unzip is available
INFO     2022-04-03 23:52:32,070 [cache.save_to_cache:28]:Skipping caching of /home/jeux/Téléchargements/tangledeep/unzip/unzip.tar.gz, already cached in /home/jeux/Téléchargements/tangledeep/unzip
DEBUG    2022-04-03 23:52:32,071 [files_box.on_file_available:93]:game is available
INFO     2022-04-03 23:52:32,071 [files_box.on_file_available:99]:All files available
INFO     2022-04-03 23:52:32,072 [installerwindow.on_files_available:463]:All files are available, continuing install
DEBUG    2022-04-03 23:52:32,073 [interpreter._iter_commands:317]:Installer command: {'extract': {'dst': '$CACHE', 'file': '$unzip'}}
INFO     2022-04-03 23:52:32,074 [cache.save_to_cache:28]:Skipping caching of /home/jeux/Téléchargements/tangledeep/gog/tangledeep_1_53a_54452.sh, already cached in /home/jeux/Téléchargements/tangledeep/gog
DEBUG    2022-04-03 23:52:32,076 [commands.extract:193]:Extracting unzip.tar.gz
DEBUG    2022-04-03 23:52:32,076 [commands.extract:197]:extracting file /home/jeux/Téléchargements/tangledeep/unzip/unzip.tar.gz to /home/jeux/.cache/lutris/installer/tangledeep
DEBUG    2022-04-03 23:52:32,100 [extract.extract_archive:123]:Extracting /home/jeux/Téléchargements/tangledeep/unzip/unzip.tar.gz to /home/jeux/.cache/lutris/installer/tangledeep
DEBUG    2022-04-03 23:52:32,103 [extract.extract_archive:175]:Finished extracting /home/jeux/Téléchargements/tangledeep/unzip/unzip.tar.gz to /home/jeux/.cache/lutris/installer/tangledeep
DEBUG    2022-04-03 23:52:32,113 [commands._killable_process:552]:Process <function extract_archive at 0x7fe054cce790> returned: ('/home/jeux/Téléchargements/tangledeep/unzip/unzip.tar.gz', '/home/jeux/.cache/lutris/installer/tangledeep')
DEBUG    2022-04-03 23:52:32,117 [commands.extract:199]:Extract done
DEBUG    2022-04-03 23:52:32,117 [interpreter._iter_commands:317]:Installer command: {'execute': {'args': '$game -d "$GAMEDIR" "data/noarch/*"', 'file': '$CACHE/unzip'}}
DEBUG    2022-04-03 23:52:32,120 [command.start:141]:LD_LIBRARY_PATH="/home/jeux/.local/share/lutris/runtime/Ubuntu-18.04-i686:/home/jeux/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:/home/jeux/.local/share/lutris/runtime/steam/i386/lib:/home/jeux/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:/home/jeux/.local/share/lutris/runtime/steam/i386/usr/lib:/home/jeux/.local/share/lutris/runtime/Ubuntu-18.04-x86_64:/home/jeux/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:/home/jeux/.local/share/lutris/runtime/steam/amd64/lib:/home/jeux/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:/home/jeux/.local/share/lutris/runtime/steam/amd64/usr/lib:/run/opengl-driver/lib:/run/opengl-driver-32/lib:/usr/lib:/usr/lib32"
DEBUG    2022-04-03 23:52:32,120 [command.start:141]:PATH="/home/jeux/.local/share/lutris/runtime/network-tools:/nix/store/wfrm0v3qbj4l44ny529m9k5wbkyjk711-xrandr-1.5.1/bin:/nix/store/sa6nc9dc4krgj7g21qdkq4a89lnp1svb-pciutils-3.7.0/bin:/nix/store/180z1lbh0kd6dxp5was1dn35rrw6lh0g-psmisc-23.4/bin:/nix/store/lgj67rjpgarz9nmvy6j31km90zsi9rvb-glxinfo-8.4.0/bin:/nix/store/jyjk9xnq3n6lv4sdh8rms9gdgcsbvsff-vulkan-tools-1.2.198.0/bin:/nix/store/q5lybdgiab3m57hy3hhn96f69fz65cnq-xboxdrv-0.8.8/bin:/nix/store/7i1f5p8bwf1drlkf1h1ppaf1l3kn7rlc-pulseaudio-14.2/bin:/nix/store/sx19fnzzz1mgjvjjl7z89p9z9y0fxcwi-p7zip-17.04/bin:/nix/store/h2vkni1hxy56n9d1qfg9dhwc47k3g8w9-xgamma-1.0.6/bin:/nix/store/iwjyqrvfl5y73xi8fpy0v17mb8m2i5zf-libstrangle-0.1.1/bin:/nix/store/lamw53y46xh8h4gdffq5hxdwmyh0p821-unzip-6.0/bin:/nix/store/8cj940z8h9aa5qpm2pgxk5hy76nmd2lk-wine-wow-7.4-staging/bin:/nix/store/6s9h42flwq6yal0k91xsf30vp0isicvc-fluidsynth-2.2.3/bin:/nix/store/jg5a4396plm3q9fpwvbl4jjqp2x0fxkp-xorg-server-1.20.13/bin:/nix/store/8wn175gx96fbwz1vq09jnalfza6ajamc-setxkbmap-1.3.2/bin:/nix/store/86i2svbkxvfg5jjz4plin0z2rdpkpz1w-xkbcomp-1.4.5/bin:/nix/store/hb1lzaisgx2m9n29hqhh6yp6hasplq1v-python3-3.9.10/bin:/nix/store/bg3hjz4xymhca5c938jkwd25da01fc7m-lutris-original-0.5.10/bin:/nix/store/pj49wryiyf5246hd975zip66hccz96zd-python3.9-distro-1.6.0/bin:/nix/store/qd9h4vgsw9sqp85qsynlnpb7j3qyag82-python3.9-nose2-0.11.0/bin:/nix/store/dg697192a69mfcbrxmxgnpfb1rv48mnc-python3.9-coverage-6.2/bin:/nix/store/hihm75l9v1rlrgzm642hb2izf2w9xh0y-cairo-1.16.0-dev/bin:/nix/store/26kn7iiginbdvkm44ss5h1hkfpgzn4fq-freetype-2.11.1-dev/bin:/nix/store/rmxjjpg43f6jssh82q29q21lzlcqzwxz-bzip2-1.0.6.0.2-bin/bin:/nix/store/2d0km1rhk9glmnk3p6g5rs8148va9wch-libpng-apng-1.6.37-dev/bin:/nix/store/0ckjw6n353xx6jxa4j26ilff46g43l37-fontconfig-2.13.94-bin/bin:/nix/store/1xb05hby252c6z0h3cma619cvrlja6x1-expat-2.4.6-dev/bin:/nix/store/k384sdx8f6gb9mqnvzspmvlsq02x701d-glib-2.72.0-dev/bin:/nix/store/pr95pmiy2wb8nv5fd69bx3pf1af5wy3j-gettext-0.21/bin:/nix/store/yqnz480v7jjlpa3kckwfmlp9fqfff02n-glib-2.72.0-bin/bin:/nix/store/5w559pg3wigwprj375afnjr7lqvzw7p5-python3.9-chardet-4.0.0/bin:/nix/store/whxdj9wxdz4p3whm2dgg9q71fnb59bp8-python3.9-charset-normalizer-2.0.12/bin:/nix/store/vyqqr2xv84v3znzww2p8n6wsqykhc6zn-python3.9-keyring-23.5.0/bin:/run/wrappers/bin:/usr/bin:/usr/sbin:/nix/store/fa138pjfv8b4sg5l80ls55ynz9g4s3p9-bash-interactive-5.1-p16/bin:/nix/store/9chsm1gqvvpqywz219s1rknfrzr5xjfz-patchelf-0.14.3/bin:/nix/store/jz10kjr0lnhynwxnrzv4kcwbjs5s0ri8-gcc-wrapper-10.3.0/bin:/nix/store/finlf6zyxhpqi8hp2p4fgxkgqvjp6l78-gcc-10.3.0/bin:/nix/store/drb5a0j2z8fh6bivl9aq15fbfmrx62kb-glibc-2.33-117-bin/bin:/nix/store/2w6011wwg5v6qxz182r4d0mmr3i4rxm1-coreutils-9.0/bin:/nix/store/55ayck4pllh4qallb1dbhbkrzwrrl02r-binutils-wrapper-2.35.2/bin:/nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin:/nix/store/2w6011wwg5v6qxz182r4d0mmr3i4rxm1-coreutils-9.0/bin:/nix/store/pr0l0cwak2dprcvvzqvli1kkpg7k9fci-findutils-4.9.0/bin:/nix/store/c5xa1n6bl34gxfmjlqgaxdyk4i583fk5-diffutils-3.8/bin:/nix/store/jdhk02s01ca02qv083f41wz8jj25gybc-gnused-4.8/bin:/nix/store/3fvq0jjjh50vh0rlqr2xc9ya8llcj113-gnugrep-3.7/bin:/nix/store/i4mm5ycljiq465ncr29mg9bf7wn18i8c-gawk-5.1.1/bin:/nix/store/4rypjibv28cbbg21apsqnnyi3k2glzjd-gnutar-1.34/bin:/nix/store/660k06nr64g3fqndivn6brmk8ly7ivdz-gzip-1.11/bin:/nix/store/rmxjjpg43f6jssh82q29q21lzlcqzwxz-bzip2-1.0.6.0.2-bin/bin:/nix/store/xgibf1z6b7zx4h1hv3v8qw0f2n1jv4pm-gnumake-4.3/bin:/nix/store/dndi916j6yxzfzzj2sma2llhrlwahq06-bash-5.1-p16/bin:/nix/store/agfkd1d9mmcniwrvh10afn054x1n8iqi-patch-2.7.6/bin:/nix/store/45j7wibbdckl5f50bs5nq7damhy8cahp-xz-5.2.5-bin/bin:/run/wrappers/bin:/home/jeux/.nix-profile/bin:/etc/profiles/per-user/jeux/bin:/nix/var/nix/profiles/default/bin:/run/current-system/sw/bin"
DEBUG    2022-04-03 23:52:32,120 [command.start:141]:PYTHONPATH="/nix/store/bg3hjz4xymhca5c938jkwd25da01fc7m-lutris-original-0.5.10/lib/lutris:/nix/store/bg3hjz4xymhca5c938jkwd25da01fc7m-lutris-original-0.5.10/bin:/nix/store/hb1lzaisgx2m9n29hqhh6yp6hasplq1v-python3-3.9.10/lib/python39.zip:/nix/store/hb1lzaisgx2m9n29hqhh6yp6hasplq1v-python3-3.9.10/lib/python3.9:/nix/store/hb1lzaisgx2m9n29hqhh6yp6hasplq1v-python3-3.9.10/lib/python3.9/lib-dynload:/nix/store/hb1lzaisgx2m9n29hqhh6yp6hasplq1v-python3-3.9.10/lib/python3.9/site-packages:/nix/store/bg3hjz4xymhca5c938jkwd25da01fc7m-lutris-original-0.5.10/lib/python3.9/site-packages:/nix/store/952jybpchzkwqm6887s1p82rqj09h2ag-python3.9-evdev-1.5.0/lib/python3.9/site-packages:/nix/store/pj49wryiyf5246hd975zip66hccz96zd-python3.9-distro-1.6.0/lib/python3.9/site-packages:/nix/store/gvsn5nknrpkb5gf2ljsw671xxw94gvjh-python3.9-lxml-4.7.1/lib/python3.9/site-packages:/nix/store/qd9h4vgsw9sqp85qsynlnpb7j3qyag82-python3.9-nose2-0.11.0/lib/python3.9/site-packages:/nix/store/dg697192a69mfcbrxmxgnpfb1rv48mnc-python3.9-coverage-6.2/lib/python3.9/site-packages:/nix/store/5cqp0x613vxqlad25w18fmzcyi0p7lzv-python3.9-six-1.16.0/lib/python3.9/site-packages:/nix/store/va4j4930kj0svadrq01pvv83k3rvbl59-python3.9-pyyaml-6.0/lib/python3.9/site-packages:/nix/store/6028k8xv7s6q76cs1cna1pxn5a8yrn1h-python3.9-pycairo-1.20.1/lib/python3.9/site-packages:/nix/store/7yysmswykdvmlmn6j1zjqg5jgap5bi2x-python3.9-pygobject-3.42.0/lib/python3.9/site-packages:/nix/store/w85hzpfx05v128aya3qslfspi4w5midk-python3.9-requests-2.27.1/lib/python3.9/site-packages:/nix/store/sp19f4jsi5azvmd7zvacaj3as9v08ini-python3.9-certifi-2021.10.08/lib/python3.9/site-packages:/nix/store/x755yb7v6faydq5f9rqhm968nszwhhiq-python3.9-idna-3.3/lib/python3.9/site-packages:/nix/store/cn0gh6jv54yhg0b0s0y7qaarh1zf3lz3-python3.9-urllib3-1.26.8/lib/python3.9/site-packages:/nix/store/h789f4mpphz0lkzsrzsr5y1zn47pqlky-python3.9-brotli-1.0.9/lib/python3.9/site-packages:/nix/store/cc2i74pg99qrqgrs2dizmmpz22d9p010-python3.9-pysocks-1.7.1/lib/python3.9/site-packages:/nix/store/5w559pg3wigwprj375afnjr7lqvzw7p5-python3.9-chardet-4.0.0/lib/python3.9/site-packages:/nix/store/ncs9snfpw195jjkyaf2q48972sbpfidl-python3.9-brotlicffi-1.0.9.2/lib/python3.9/site-packages:/nix/store/ykg29kij732bj9vic5vsw993s9wq1686-python3.9-pycparser-2.21/lib/python3.9/site-packages:/nix/store/8nhm05fvzscamaqmcpqfkqna6lwdb4m1-python3.9-cffi-1.15.0/lib/python3.9/site-packages:/nix/store/whxdj9wxdz4p3whm2dgg9q71fnb59bp8-python3.9-charset-normalizer-2.0.12/lib/python3.9/site-packages:/nix/store/3vgimz37kmdpnaqmffwvlgngz3ac5376-python3.9-Pillow-9.0.1/lib/python3.9/site-packages:/nix/store/c433m2jsrgx2a51m5qslhv3nqnqg73xm-python3.9-olefile-0.46/lib/python3.9/site-packages:/nix/store/y0w267yv7x4yv27clc2bpgw9vxgsds3n-python3.9-defusedxml-0.7.1/lib/python3.9/site-packages:/nix/store/4zwk5cfavn6792d0xzp8s2m5v0iw7f54-python3.9-dbus-python-1.2.18/lib/python3.9/site-packages:/nix/store/vyqqr2xv84v3znzww2p8n6wsqykhc6zn-python3.9-keyring-23.5.0/lib/python3.9/site-packages:/nix/store/r5mhzhpgr5daw9yl4x4jf7l2qswh5737-python3.9-importlib-metadata-4.11.0/lib/python3.9/site-packages:/nix/store/6nb8g6wwm0kshjqwqhqkhvzicqcv18ml-python3.9-toml-0.10.2/lib/python3.9/site-packages:/nix/store/3v32annsicl1ffl4g6ihm4l0jwlzr92s-python3.9-zipp-3.7.0/lib/python3.9/site-packages:/nix/store/zyvmgzk0acldhf6sxb8nh9xslg5q0j1y-python3.9-more-itertools-8.12.0/lib/python3.9/site-packages:/nix/store/6xnp0gpc4hn4j2sks0rxhxrc7w80s0dp-python3.9-jeepney-0.7.1/lib/python3.9/site-packages:/nix/store/pkv6c7p7sz15v6sc63sm1ipacpfczimn-python3.9-secretstorage-3.3.1/lib/python3.9/site-packages:/nix/store/9dlrj1saa8k0mpfc3v54ww2hypsasbpa-python3.9-packaging-21.3/lib/python3.9/site-packages:/nix/store/rv3dbmbbb7h9x1rs488fq4d2d0p9cngg-python3.9-pyparsing-3.0.6/lib/python3.9/site-packages:/nix/store/1ak7jkgjyyqxwy13cm2qzfvdyy8d47nc-python3.9-cryptography-36.0.0/lib/python3.9/site-packages:/nix/store/hj279gd5im2xr5nwgmvfx1bxxb0n3fhd-python3.9-python-magic-0.4.25/lib/python3.9/site-packages"
DEBUG    2022-04-03 23:52:32,120 [command.start:141]:LUTRIS_GAME_UUID="4bf3d506-dbc2-4040-b76a-32c8cd285e6a"
lutris-wrapper: /home/jeux/.cache/lutris/installer/tangledeep/unzip
Started initial process 123 from /home/jeux/.cache/lutris/installer/tangledeep/unzip /home/jeux/Téléchargements/tangledeep/gog/tangledeep_1_53a_54452.sh -d /home/jeux/GOG Games/gog/tangledeep data/noarch/*
Start monitoring process.
/home/jeux/.cache/lutris/installer/tangledeep/unzip: error while loading shared libraries: libbz2.so.1.0: cannot open shared object file: No such file or directory
Monitored process exited.
Initial process has exited (return code: 32512)
All processes have quit
Exit with return code 32512
DEBUG    2022-04-03 23:52:32,278 [command.on_stop:195]:Process 99 has terminated with code 32512
DEBUG    2022-04-03 23:52:33,126 [commands._monitor_task:449]:Return code: 32512
ERROR    2022-04-03 23:52:33,127 [errors.__init__:18]:Command exited with code 32512
None
INFO     2022-04-03 23:52:36,502 [interpreter.revert:374]:Cancelling installation of Tangledeep
DEBUG    2022-04-03 23:52:36,503 [system.remove_folder:229]:Removing folder /home/jeux/GOG Games/gog/tangledeep
DEBUG    2022-04-03 23:52:36,504 [system.remove_folder:229]:Removing folder /home/jeux/.cache/lutris/installer/tangledeep
DEBUG    2022-04-03 23:52:36,505 [application.on_app_window_destroyed:350]:Removed window InstallerWindow1703478259
INFO     2022-04-03 23:52:37,608 [application.do_shutdown:885]:Shutting down Lutris
```

### System Information

```shell
[System]
OS:              NixOS 22.05 quokka
Arch:            x86_64
Kernel:          5.17.1
Desktop:         KDE
Display Server:  x11

[CPU]
Vendor:          GenuineIntel
Model:           Intel(R) Core(TM) i5-7300U CPU @ 2.60GHz
Physical cores:  2
Logical cores:   4

[Memory]
RAM:             7.6 GB
Swap:            5.4 GB

[Graphics]
Vendor:          Intel
OpenGL Renderer: Mesa Intel(R) HD Graphics 620 (KBL GT2)
OpenGL Version:  4.6 (Compatibility Profile) Mesa 21.3.7
OpenGL Core:     4.6 (Core Profile) Mesa 21.3.7
OpenGL ES:       OpenGL ES 3.2 Mesa 21.3.7
Vulkan:          Supported
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by rapenne-s** [comment]
My temporary poor fix:

- try to install the game and keep the downloaded files in a cache directory
- rm the `unzip.tar.gz` file in the cache
- create a `unzip.tar.gz` archive using `unzip` from my $PATH
- run installation again

**Comment by strycore** [comment]
Any script still using unzip (or innoextract as well, for that matter) needs to be fixed. We ship the necessary tools as part of the lutris runtime. While those scripts are being fixed or deleted, the auto-generated one is the easiest solution.

Now, the tools we ship might or might not be compatible with the Nix ABI. 
Since we're adding Flatpak support, I'm considering having a system in place where we depend on locally available tools and libraries instead of having to ship them in a runtime.
This doesn't have to be specific to Flatpak and can be expanded to Nix.

**Comment by strycore** [comment]
This can be merged with #4852 

I'll likely make a script on the website to automate the migration.
